packages/cardhost/tests/helpers/card-helpers.js

```diff
@@ -28,6 +33,7 @@ export async function createCards(args) {
 }
 
 export async function addField(name, type, isEmbedded, position) {
+  await dragAndDropField(0);
```

Comment:
we are still using the old way to add a field in this test. I was thinking that the drag and drop was going to replace the old way of adding fields. so that you didn't need to click on the `[data-test-field-creator-add-field-btn]` button to add a field.

Reply:
@habdelra this is basically a stopgap between the current way and what you are mentioning. We could skip the step of using the `<FieldCreator>` if we fake it and give the field some default name, i.e. `field1` or whatever, but until we have a right edge, there won't be a way to edit the `name` or other attributes of the field.

Reply:
I'm not really sure what you are testing here then. like if I were to comment out that line, the test will still pass

Reply:
probably you'll have to make some kind of throw away field to capture the name and embedded check box too...

Reply:
or continue to use the FieldCreator component for setting the name and embedded fields and ket right edge work deal with refactoring that

Reply:
Another idea is to use the `<FieldRenderer>` just put it on the page someone for the right edge person to cleanup. For the `<FieldRenderer you could pass to it the field that you created at dragend event, and use the `@mode=schema` in the `<FieldRenderer>`--this is probably the most straight forward approach using an already built mechanism for capturing field names.

Reply:
Ultimately, I'd like to not go backwards in terms of functionality. If you are not sure how to move forwards here then we can pair on it, or if you prefer I can add a commit to this branch to do that.